app/controllers/miq_policy_controller/policy_profiles.rb

```diff
@@ -5,7 +5,7 @@ def profile_edit
     case params[:button]
     when "cancel"
       @edit = nil
-      @profile = MiqPolicySet.find_by_id(session[:edit][:profile_id]) if session[:edit] && session[:edit][:profile_id]
+      @profile = MiqPolicySet.find(session[:edit][:profile_id]) if session[:edit] && session[:edit][:profile_id]
```

Comment:
This will raise an exception and it might be correct that it does (due to the test).
But the line below is crazy.
What about `@profile.blank?`  ?